Gemfile

```diff
@@ -99,6 +101,11 @@ gem "websocket-driver",               "~>0.6.3"
 gem "foreman_api_client",             ">=0.1.0",   :require => false, :git => "https://github.com/ManageIQ/foreman_api_client.git", :branch => "master"
 gem "ruport",                         "=1.7.0",                       :git => "https://github.com/ManageIQ/ruport.git", :tag => "v1.7.0-3"
 
+gem "uuid"
+
+# To Lenovo provider
+gem "xclarity_client"
```

Comment:
We weren't sure how the dependencies cascaded.  We can place this gem in manageiq-providers-lenovo and remove it from here.